### ISSUE TITLE
Prison camps

### ISSUE THREAD

**Issue by joereddington** [eQualityTime/Public#105]
Prison A 

 March (16th, 17th, 18th, 19th & 20th March for delivery of sessions & 8th April for the family

Prison B 

WC 27th April. The illustration day would be 1st May & book launch would be 4th June.

 

WC 6th July. The illustration day would be 10th July & book launch would be 4th August.

 

WC 19th October. The illustration day would be 23rd October & book launch would be 19th November.

WC 11th January 2021. The illustration day would be 15th January & book launch would be 18th February.

**Comment by joereddington** [comment]
We're currently negociating over email about moving this forward.

**Comment by joereddington** [comment]
* Prison A is done, although I haven't got the files yet. (I've got a reminder to email) 
* I've checked in about bookings with Novus to see where they are, I'll badger again about deposit shortly. 
* One of the things to think about is that I need to nominate 
   * Back up dates to the main dates that named staff can do. 
   * Back up staff to the back up staff. There probably needs to be four named staff that can get in touch at any given time.
